### PR TITLE
[explorer/nodewatch] fix: linting with invalid name

### PR DESCRIPTION
**Pull request by Wayonb**
problem: linting is failing with invalid name
solution: move the disable of invalid name to the top of the file.

### CODE REVIEW

File: explorer/nodewatch/tests/test_HeightChartBuilder.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# pylint: disable=invalid-name
```
Comment (Wayonb) [inline explorer/nodewatch/tests/test_HeightChartBuilder.py:1]: Not sure why this doesnt fail for all run though 🤔 
https://jenkins.symboldev.com/job/Symbol/job/generated/job/product/job/nodewatch/job/dev/

**Comment by Jaguar0625** [comment]
```
21:26:31  ************* Module _NetworkConnector
21:26:31  tests/test_NetworkConnector.py:1:0: C0103: Module name "_NetworkConnector" doesn't conform to '((test_)?([A-Za-z][a-zA-Z0-9_]+)|(__main__))$' pattern (invalid-name)
```

why is it detecting module name as `_NetworkConnector` instead of (valid) `test_NetworkConnector` ?

**Comment by Wayonb** [comment]
yea, probably.  What I am stuck on is why only this project.  All the python projects use the same version.
Also downgrading would mean for all python projects, which is probably fine.

**Comment by Wayonb** [comment]
I created a bug on pylint https://github.com/pylint-dev/pylint/issues/8518

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**